examples/tutorials/forced_alignment_for_multilingual_data_tutorial.py

```diff
@@ -0,0 +1,905 @@
+    torch.hub.load_state_dict_from_url(
+        "https://dl.fbaipublicfiles.com/mms/torchaudio/ctc_alignment_mling_uroman/model.pt"
+    )
+)
+model.eval()
+
+
+def get_emission(waveform):
+    # NOTE: this step is essential
+    waveform = torch.nn.functional.layer_norm(waveform, waveform.shape)
+
+    emissions, _ = model(waveform)
+    emissions = torch.log_softmax(emissions, dim=-1)
+    emission = emissions[0].cpu().detach()
+
+    # Append the extra dimension corresponding to the <star> token
+    extra_dim = torch.zeros(emissions.shape[0], emissions.shape[1], 1)
+    emissions = torch.cat((emissions.cpu(), extra_dim), 2)
+    emission = emissions[0].detach()
+    return emission, waveform
+
+
+# Construct the dictionary
+# '@' represents the OOV token, '*' represents the <star> token.
+# <pad> and </s> are fairseq's legacy tokens, which're not used.
+dictionary = {
+    "<blank>": 0,
+    "<pad>": 1,
+    "</s>": 2,
+    "@": 3,
+    "a": 4,
+    "i": 5,
+    "e": 6,
+    "n": 7,
+    "o": 8,
+    "u": 9,
+    "t": 10,
+    "s": 11,
+    "r": 12,
+    "m": 13,
+    "k": 14,
+    "l": 15,
+    "d": 16,
+    "g": 17,
+    "h": 18,
+    "y": 19,
+    "b": 20,
+    "p": 21,
+    "w": 22,
+    "c": 23,
+    "v": 24,
+    "j": 25,
+    "z": 26,
+    "f": 27,
+    "'": 28,
+    "q": 29,
+    "x": 30,
+    "*": 31,
+}
+
+
+######################################################################
+# Before aligning the speech with transcripts, we need to make sure
+# the transcripts are already romanized. Here are the BASH commands
+# required for saving raw transcript to a file, downloading the uroman
+# romanizer and using it to obtain romanized transcripts, and PyThon
+# commands required for further normalizing the romanized transcript.
+#
+
+# %%
+# .. code-block:: bash
+#
+#    %%bash
+#    Save the raw transcript to a file
+#    echo 'raw text' > text.txt
+#    git clone https://github.com/isi-nlp/uroman
+#    uroman/bin/uroman.pl < text.txt > text_romanized.txt
+#
+
+######################################################################
+# .. code-block:: python
+#
+#    import re
+#    def normalize_uroman(text):
+#        text = text.lower()
+#        text = text.replace("’", "'")
+#        text = re.sub("([^a-z' ])", " ", text)
+#        text = re.sub(' +', ' ', text)
+#        return text.strip()
+#
+#    file = "text_romanized.txt"
+#    f = open(file, "r")
+#    lines = f.readlines()
+#    text_normalized = normalize_uroman(lines[0].strip())
+#
+
+
+######################################################################
+# German example:
+# ~~~~~~~~~~~~~~~~
+
+text_raw = (
+    "aber seit ich bei ihnen das brot hole brauch ich viel weniger schulze wandte sich ab die kinder taten ihm leid"
+)
+text_normalized = (
+    "aber seit ich bei ihnen das brot hole brauch ich viel weniger schulze wandte sich ab die kinder taten ihm leid"
+)
+SPEECH_FILE = torchaudio.utils.download_asset("tutorial-assets/10349_8674_000087.flac")
```

Comment:
`SPEECH_FILE` in other tutorials are all capped like global variables to indicate that it is used throughout the tutorial, not just in one section.

Here, and from this point, SPEECH_FILE is overwritten multiple times, which looks strange. Can you lower case?